File: src/test/java/io/github/binout/soccer/domain/player/PlayerTest.java
package io.github.binout.soccer.domain.player;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class PlayerTest {

    @Test
    public void email_with_blank_is_like_optional_empty() {
        assertThat(new Player("sylvain", "").email()).isEmpty();
    }
}

File: src/test/kotlin/io/github/binout/soccer/domain/player/PlayerTest.kt
package io.github.binout.soccer.domain.player

import org.junit.jupiter.api.Test

import org.assertj.core.api.Assertions.assertThat

class PlayerTest {

    @Test
    fun email_with_blank_is_like_optional_empty() {
        assertThat(Player("sylvain", "").email()).isEmpty
    }
}
